bokeh/command/subcommands/serve.py

```diff
@@ -727,6 +727,12 @@ class Serve(Subcommand):
         )),
     )
 
+    def customize_kwargs(self, args: argparse.Namespace, server_kwargs: Dict[str, Any]) -> Dict[str, Any]:
+        '''Allows subclasses to return kwargs to be merged with server_kwargs.
+
+        '''
+        return {}
```

Comment:
I actually had in mind that `customize_kwargs` should make a copy, update the copy, then return it. i.e. so that the default implementation ought to return `dict(server_kwargs)`  (The docstring should be clear about this)